tickets/src/utils/dates.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This function returns the future days, months and up to 3 years based on a current
+ * date.
+ * @param {*} now
+ * @param {*} year
+ * @param {*} month
+ */
+export const getDaysMonthsAndYears = (now, year, month) => {
+  let monthNames = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ]
+
+  // Years are the next 3 years
+  const years = [
+    now.getFullYear(),
+    now.getFullYear() + 1,
+    now.getFullYear() + 2,
+  ].map(d => d.toString())
+
+  let months = [...monthNames]
+
+  if (year === `${now.getFullYear()}`) {
+    months = months.slice(now.getMonth(), months.length)
+  }
+
+  let days = Array.from(new Array(31), (val, index) => (index + 1).toString())
+  if (['Apr', 'Jun', 'Sep', 'Nov'].indexOf(month) > -1) {
+    // only 30 days!
+    days.pop()
+  }
+  if (month === 'Feb') {
+    // bissextile years?
+    if (year % 4 === 0) {
```

Comment:
This isn't quite correct, leap years aren't only the ones divisible by 4.

Reply:
In addition, every 100 years is not a leap year (although we will not live to see the next one probably)

Reply:
Ha! Yeah I assumed this would be good enough but I guess I'll appluy this:

```if (year is not divisible by 4) then (it is a common year)
else if (year is not divisible by 100) then (it is a leap year)
else if (year is not divisible by 400) then (it is a common year)
else (it is a leap year) ```